src/main/java/org/example/OrderFile.java

```diff
@@ -0,0 +1,19 @@
+package org.example;
+
+import java.io.FileWriter;
+
+public class OrderFile {
+    public static void creatFileOrder(String name, double quantity, int amount) {
```

Comment:
Let's pass `Goods` as a parameter instead of 3 these params, since in Main class you've already created this obkect